cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -601,7 +601,9 @@ and then restart CDAP.
 
 .. highlight:: console
 
-1. Stop all CDAP processes::
+1. Stop all Flows, Services, and Programs in all your applications.
```

Comment:
Flows and services are programs right? Is this supposed to say MapReduce Programs?